testCourse/questions/orderBlocks/question.html

```diff
@@ -48,3 +48,35 @@
 </pl-order-blocks>
 
 <!-- TODO add more to cover the rest of the possible option configurations. -->
+
+
```

Comment:
```suggestion
```

Or move this back to the bottom of this file, and ideally list which configurations aren't already covered.

Reply:
Listing which ones aren't already covered is half the work of actually doing it :sweat_smile:. I can just remove it for now.